src/routes/PhotoRouter.ts

```diff
@@ -20,6 +20,7 @@ class PhotoRouter implements IRouter {
   initializeRoutes() {
     this.router.get(`${this.path}/:id(\\d+)`, this.photoController.getById);
     this.router.post(this.path, myValidator(PhotoCreateValidator, true), this.photoController.create);
+    this.router.patch(`${this.path}/:id(\\d+)`, myValidator(PhotoCreateValidator, true), this.photoController.update);
```

Comment:
We should use an appropriate validator for updates, for example, we want to allow `isPublic` to be updatable and we want to allow empty fields that are not being updated as well.